### ISSUE TITLE
Bug - Buffer error on ubuntu

### ISSUE THREAD

**Issue by drj-io** [rackfx/Node-WAV-File-Info#3]
Console:

```
app-0 about to read  /home/ubuntu/webapp/.tmp/uploads/54af9211-7140-4a61-a27e-efed9cb483e1.wav
app-0
app-0 /home/ubuntu/webapp/node_modules/wav-file-info/wav-file-info.js:48
app-0           read_result[read[0]] = buffer.readIntLE(pointer, read[2])
app-0  app-0  app-0  app-0  app-0  app-0  app-0  app-0  app-0  app-0  app-0  app-0  app-0                             ^
app-0 TypeError: Object RIFF��
PM2 App name:app id:0 exited with code 8
PM2 Starting execution sequence in -fork mode- for app name:app id:0
PM2 App name:app id:0 online
```

**Comment by drj-io** [comment]
This is still an issue.

npm install wav-file-info

creates the error on ubuntu

but git clone does not...

npm install wav-file-info
works on OSx

**Comment by drj-io** [comment]
Resolved on 0.0.5
